apps/api-site-test/test/integration/update-site-metadata.spec.ts

```diff
@@ -14,88 +16,106 @@ describe('Update Site Metadata', () => {
   let api: TestAgent
   let resetService: SiteApiResetService
   let adminAuth: string
+  let ownerAuth: string
   let payload: IUpdateSiteMetadataApiRequest
   let siteId: string
-  let ownerId: string
 
   beforeAll(() => {
     api = supertest(testConfig.get('apiUrl'))
     adminAuth = adminAuthHeader()
+    ownerAuth = ownerAuthHeader('903b3c28-deaa-45dc-a43f-511fe965d34e')
     resetService = new SiteApiResetService('http://127.0.0.1:3100', adminAuth, SITE_SEEDS)
   })
 
   beforeEach(async () => {
     await resetService.reset()
-    ownerId = '903b3c28-deaa-45dc-a43f-511fe965d34e'
     siteId = '6d2c8359-6094-402c-bcbb-37202fd7c336'
     payload = { domains: ['update-subdomain.dev.pubstud.io', 'shop.abc.com'] }
   })
 
-  it('updates site and verifies result when requester is admin', async () => {
-    await api
-      .patch(`${testEndpoint}/${siteId}`)
-      .set('Authorization', adminAuth)
-      .send(payload)
-      .expect(200)
-
-    // Verify domains are updated
-    const getResponse = await api
-      .get(`/api/sites/${siteId}/domains`)
-      .set('Authorization', adminAuth)
-      .expect(200)
-    const getBody: IGetSiteDomainsApiResponse = getResponse.body
-
-    expect(getBody.domains).toEqual(payload.domains)
-  })
+  describe('when requester is admin', () => {
+    it('returns 200 status and message when update metadata', async () => {
+      await api
+        .patch(`${testEndpoint}/${siteId}`)
+        .set('Authorization', adminAuth)
+        .send(payload)
+        .expect(200)
 
-  it('disables site and verifies it cannot be retrieved', async () => {
-    payload = {
-      disabled: true,
-    }
-    await api
-      .patch(`${testEndpoint}/${siteId}`)
-      .set('Authorization', adminAuth)
-      .send(payload)
-      .expect(200)
-
-    // Verify owner can no longer access
-    const ownerAuth = ownerAuthHeader(ownerId)
-    await api.get(`/api/sites/${siteId}`).set('Authorization', ownerAuth).expect(403, {
-      code: 'SiteDisabled',
-      message: 'Forbidden',
-      status: 403,
+      // Verify domains are updated
+      const getResponse = await api
+        .get(`/api/sites/${siteId}/domains`)
+        .set('Authorization', adminAuth)
+        .expect(200)
+      const getBody: IGetSiteDomainsApiResponse = getResponse.body
+
+      expect(getBody.domains).toEqual(payload.domains)
     })
 
-    // Verify anonymous can no longer access
-    await api.get('/api/sites/current').set('Host', 'www.myblog.org').expect(403, {
-      code: 'SiteDisabled',
-      message: 'Forbidden',
-      status: 403,
+    it('returns 200 status and message when update site type', async () => {
+      payload.site_type = SiteType.Paid1
```

Comment:
I changed this to use the enum, and noticed I forgot to migrate a type file from the private repo, so I added it!